datadog_lambda/wrapper.py

```diff
@@ -181,6 +185,10 @@ def _after(self, event, context):
                 flush_extension()
 
```

Comment:
Can we enable this feature in the integration tests as well for improved test coverage?

Reply:
It appears that we don't care about span tags in our integration tests, because the integration tests are unaffected by this ENV variable (off or on). We can do this during an engineering sprint, perhaps?

Reply:
Hmm, that's weird, the integration tests are supposed to capture span tags. Let's dig into this a bit more before merging to make sure nothing is wrong. Happy to chat on Zoom about this if you want.